backend/tests/test_graphql_api/test_application/test_filtering.py

```diff
@@ -232,14 +232,16 @@ def test_application__filter__by_text_search__section_name(graphql):
         organisation_name="",
         contact_person_first_name="",
         contact_person_last_name="",
-        user=None,
+        user__first_name="",
+        user__last_name="",
```

Comment:
Would it be better to have this in the factory method as default values?

Reply:
Nah, these are special case tests where we need to know that the other values don't contribute to the text search.